packages/bs-ethereum/src/graphql.ts

```diff
@@ -0,0 +1,281 @@
+import { gql } from '@urql/core'
```

Comment:
Wouldn't be better to use that code generator to generate all these types? (`graphql-codegen`)

Reply:
But well, since you already did by hand and using the generator would probably make you refactor everything. I think you can keep it this way.